airflow/datasets/__init__.py

```diff
@@ -55,3 +55,13 @@ def __eq__(self, other):
 
     def __hash__(self):
         return hash(self.uri)
+
+    def __or__(self, other):
+        from airflow.models.dataset import DatasetAny
```

Comment:
May I know why do we want to import here? Is there anything expensive in `airflow.models.dataset`

Reply:
The code is changed  now